great_expectations/datasource/new_datasource.py

```diff
@@ -421,6 +423,12 @@ def config(self) -> dict:
         return copy.deepcopy(self._datasource_config)
 
 
+NOT_FLUENT_ERROR_MSG: str = (
+    "Not a 'Fluent' Datasource. Please refer to `0.15` docs https://docs.greatexpectations.io/docs/0.15.50/"
+    "\n or recreate your datasource with our new 'Fluent' API https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_overview/"
+)
```

Comment:
Could use some guidance around the wording of this error.

Reply:
I think it looks good, but how about this tweak?:
```suggestion
NOT_FLUENT_ERROR_MSG: str = (
    "Not a 'Fluent' Datasource. Please refer to `0.15` docs for info on non 'Fluent' Datasources: https://docs.greatexpectations.io/docs/0.15.50/"
    "\n or recreate your datasource with our new 'Fluent' API: https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_overview/"
)
```